proxy/http/HttpTransact.h

```diff
@@ -956,6 +956,8 @@ class HttpTransact
       return std::min(cur_parent_max_attempts, txn_conf->parent_connect_attempts) - 1;
     }
 
+    bool is_method_stats_incremented = false;
```

Comment:
This feels like an odd place to add this member. Can you move it up where all the other bool members are?

Reply:
Fixed.